src/buildOptionsList.js

```diff
@@ -4,13 +4,24 @@ import Icon from "react-native-vector-icons/AntDesign";
 
 import getDeviceInfo from "./getDeviceInfo";
 
-export const buildList = (items, setVisible, mail, reportIcon, dismissIcon) => {
+export const buildOptionsList = (
+  items,
+  setVisible,
+  email,
+  reportIcon,
+  dismissIcon,
+  reportFunction,
+) => {
   const handleReport = () => {
     const info = getDeviceInfo();
-    if (mail === undefined) {
-      alert("You didn't provide an email address as prop")
+
+    if (email === undefined) {
```

Comment:
I think that we decided to allow both methods of reporting. If an email is present, then send the email. If the report function is there, then call it, even if the email is also present.